sktime/libs/momentfm/models/moment.py

```diff
@@ -19,6 +21,9 @@
     get_huggingface_model_dimensions,
 )
 
+if platform.system() == "Darwin":
```

Comment:
I would not put this here - or anywhere where the environment is manipulated on import of something else, as an unexpected side effect!